[2023-06-27T14:47:26.666+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-27T14:47:26.668+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once'
[2023-06-27T14:47:26.703+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-27T14:47:26.704+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once'
[2023-06-27T14:47:26.922+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.load_seed_todl_once manual__2023-06-27T14:47:24.611246+00:00 [queued]>
[2023-06-27T14:47:26.937+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Setup.load_seed_todl_once manual__2023-06-27T14:47:24.611246+00:00 [queued]>
[2023-06-27T14:47:26.938+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-27T14:47:26.956+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): load_seed_todl_once> on 2023-06-27 14:47:24.611246+00:00
[2023-06-27T14:47:26.961+0000] {standard_task_runner.py:57} INFO - Started process 2321 to run task
[2023-06-27T14:47:26.965+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Setup', 'load_seed_todl_once', 'manual__2023-06-27T14:47:24.611246+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpjvsj14j9']
[2023-06-27T14:47:26.968+0000] {standard_task_runner.py:85} INFO - Job 303: Subtask load_seed_todl_once
[2023-06-27T14:47:27.019+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once permission to 509
[2023-06-27T14:47:27.020+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Setup/run_id=manual__2023-06-27T14:47:24.611246+00:00/task_id=load_seed_todl_once'
[2023-06-27T14:47:27.023+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Setup.load_seed_todl_once manual__2023-06-27T14:47:24.611246+00:00 [running]> on host eb22b15bae36
[2023-06-27T14:47:27.172+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Setup' AIRFLOW_CTX_TASK_ID='load_seed_todl_once' AIRFLOW_CTX_EXECUTION_DATE='2023-06-27T14:47:24.611246+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-27T14:47:24.611246+00:00'
[2023-06-27T14:47:27.175+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-27T14:47:27.176+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt seed --profiles-dir .']
[2023-06-27T14:47:27.187+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T14:47:30.905+0000] {subprocess.py:93} INFO - [0m14:47:30  Running with dbt=1.5.1
[2023-06-27T14:47:38.175+0000] {subprocess.py:93} INFO - [0m14:47:38  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-27T14:47:38.177+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2023-06-27T14:47:38.177+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day_lu
[2023-06-27T14:47:38.214+0000] {subprocess.py:93} INFO - [0m14:47:38  Found 3 models, 0 tests, 0 snapshots, 0 analyses, 456 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-27T14:47:38.217+0000] {subprocess.py:93} INFO - [0m14:47:38
[2023-06-27T14:47:39.283+0000] {subprocess.py:93} INFO - [0m14:47:39  Concurrency: 200 threads (target='dev')
[2023-06-27T14:47:39.284+0000] {subprocess.py:93} INFO - [0m14:47:39
[2023-06-27T14:47:39.340+0000] {subprocess.py:93} INFO - [0m14:47:39  1 of 1 START seed file ***.time_of_day_lu .................................. [RUN]
[2023-06-27T14:47:41.664+0000] {subprocess.py:93} INFO - [0m14:47:41  1 of 1 OK loaded seed file ***.time_of_day_lu .............................. [[32mINSERT 24[0m in 2.32s]
[2023-06-27T14:47:41.773+0000] {subprocess.py:93} INFO - [0m14:47:41
[2023-06-27T14:47:41.774+0000] {subprocess.py:93} INFO - [0m14:47:41  Finished running 1 seed in 0 hours 0 minutes and 3.55 seconds (3.55s).
[2023-06-27T14:47:41.856+0000] {subprocess.py:93} INFO - [0m14:47:41
[2023-06-27T14:47:41.858+0000] {subprocess.py:93} INFO - [0m14:47:41  [32mCompleted successfully[0m
[2023-06-27T14:47:41.859+0000] {subprocess.py:93} INFO - [0m14:47:41
[2023-06-27T14:47:41.861+0000] {subprocess.py:93} INFO - [0m14:47:41  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2023-06-27T14:47:42.534+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-27T14:47:42.559+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Setup, task_id=load_seed_todl_once, execution_date=20230627T144724, start_date=20230627T144726, end_date=20230627T144742
[2023-06-27T14:47:42.609+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-27T14:47:42.635+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
